01-JS-DOM-APIs/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link href="style.css" rel="stylesheet" type="text/css" >
+    <script type="text/javascript" src="javascript.js"> </script>
+    <meta charset="utf-8">
+
+    <title>JS</title>
+  </head>
+     <body onload="setTimeout(show, 1000);">
+
+    <section id="all">
+      <section>
+        <p> This is random content</p>
+        And this too.
+        <br>
```

Comment:
This is great, but try to style your markup with css to apply the concepts you learned =)
Alson, the <br/> tag should be self closing